.changeset/modern-berries-deliver.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/date-input': minor
```

Comment:
Could you please update the changeset to include all the components that have been updated? 🙏

Reply:
@CarlosCortizasCT sorry about that, edited and pushed the updated changeset now